draft-xchacha-rfc-draft00.txt

```diff
@@ -149,7 +151,10 @@ Internet-Draft                                            September 2018
 
    By generating a subkey from a 128-bit nonce and the key, a reuse of
    only the latter 64 bits of the nonce isn't security-affecting, since
-   the key (and thus, keystream) will be different.
+   the key (and thus, keystream) will be different.  Additionally a 
+   re-use of only the first 128 bits of the nonce isn't 
+   security-affecting, as the nonce derived from the latter 64 bits
+   is different.
```

Comment:
Thought it may be important to explicitly mention that the two parts of the nonce can be re-used without affecting the other. It was a little bit confusing before, as it could have been mis-interpreted as the last 64 bits of the nonce didn't affect the keystream at all. (And you'd in effect have a 128 bit nonce)

Reply:
Good point. This should be made clearer.